src/components/bulkConcepts/component/SearchBar.jsx

```diff
@@ -17,6 +17,8 @@ const SearchBar = ({
           onChange={handleChange}
           placeholder="search concept"
           required
+          pattern="^.{3,}$"
```

Comment:
Can we avoid duplicating the pattern and message?

Reply:
@dkayiwa I have declared the pattern and the message/warning in the `generalSearchReducer` and passed it through props.
I've also provided the required mock data for all tests that required those new attributes.

Reply:
@dkayiwa I would have to change it in just one place actually. All those other times are just mock data for tests. I found that the way tests were written, mock data must be passed for all tests that require it. That means all tests that have Dictionary Search or Concepts Search. 

*For example:* Mock data is repeated (as props) for tests within this file: `https://github.com/openmrs/openmrs-ocl-client/blob/master/src/tests/Dashboard/container/DictionaryDisplay.test.jsx`

Reply:
If you changed it in that one place, would the tests still be testing 4 characters or 3?

Reply:
@dkayiwa I understand the concern and have therefore created a single source of truth for tests as well. The pattern and the message are now declared only ONCE in the entire application.